main.go

```diff
@@ -170,6 +170,16 @@ func configureLoginCommand(app *kingpin.Application, ui Ui, g *globalFlags) {
 		Short('t').
 		StringVar(&input.MfaToken)
 
+	cmd.Flag("federation-token-ttl", "Expiration time for aws console session").
+		Default("12h").
+		OverrideDefaultFromEnvar("AWS_FEDERATION_TOKEN_TTL").
+		Short('f').
+		DurationVar(&input.FederationTokenDuration)
+
+	cmd.Flag("assume-role-ttl", "Expiration time for aws assumed role").
+		Default("15m").
```

Comment:
This may have been discussed previously - the aws cli defaults to 1 hour, should we match it?

> The temporary security credentials are valid for the duration that you specified when calling assume-role , which can be from 900 seconds (15 minutes) to a maximum of 3600 seconds (1 hour). The default is 1 hour.

\- http://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html